src/lib/Scenes/Consignments/v2/State/navigationModel.tsx

```diff
@@ -104,8 +105,8 @@ export const navigationModel: NavigationModel = {
     })
   }),
 
-  navigateToArtworkDetail: action(state => {
-    SwitchBoard.presentNavigationViewController(state.navViewRef.current, "/my-collection/artwork-detail")
+  navigateToArtworkDetail: action((state, payload) => {
```

Comment:
Maybe `payload` -> `artworkID`

Reply:
No preference on my end. I can make these changes.
```suggestion
  navigateToArtworkDetail: action((state, artworkID) => {
```